server/controllers/notifications.js

```diff
@@ -34,8 +33,7 @@ exports.getUnReadNotifications = asyncHandler(async (req, res) => {
     );
 
     return res.status(200).json({
-      result: unReadNotifications.length,
-      data: unReadNotifications,
+      success: unReadNotifications,
```

Comment:
Following the interface format:
```suggestion
      success: {
        unReadNotifications: unReadNotifications
      }
```